README.md

```diff
@@ -24,7 +24,7 @@ Reloads the menus and settings
 Execute any action for the player
 
 ## Configuration
-Visit our [wiki](https://github.com/help-chat/DeluxeMenus/wiki) to get info about how to make GUIs, items, etc.  
+Visit our [wiki](https://wiki.helpch.at/clips-plugins/deluxemenusi) to get info about how to make GUIs, items, etc.  
```

Comment:
Should be `https://wiki.helpch.at/clips-plugins/deluxemenus`. There's an extra `i` in there

Reply:
oopsie